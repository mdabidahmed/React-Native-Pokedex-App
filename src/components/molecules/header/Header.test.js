import React from 'react';
import HeaderComponent from './Header';

import renderer from 'react-test-renderer';

describe('HeaderComponent', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<HeaderComponent description="Test description" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
