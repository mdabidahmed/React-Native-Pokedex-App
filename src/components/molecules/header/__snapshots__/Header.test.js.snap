// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "paddingHorizontal": 16,
    }
  }
>
  <View
    style={
      {
        "paddingVertical": 8,
      }
    }
  >
    <Image
      accessibilityLabel="Pokemon Logo image"
      source={
        {
          "uri": "https://in.portal-pokemon.com/img/common/logo.png",
        }
      }
      style={
        {
          "height": 50,
          "resizeMode": "contain",
          "width": 150,
        }
      }
    />
  </View>
  <View
    style={
      {
        "alignItems": "center",
      }
    }
  >
    <View
      style={
        {
          "borderBottomColor": "#1A237E",
          "borderBottomWidth": 0.5,
          "marginVertical": 4,
        }
      }
    />
  </View>
  <Text
    style={
      {
        "color": "#11114e",
        "fontSize": 16,
        "fontWeight": "600",
        "marginBottom": 4,
        "paddingLeft": 8,
      }
    }
  >
    Test description
  </Text>
</RCTSafeAreaView>
`;
